frontend/src/Editor/QueryManager/QueryEditors/TooljetDatabase/DropDownSelect.jsx

```diff
@@ -24,40 +24,35 @@ const DropDownSelect = ({
   showPlaceHolder = false,
   highlightSelected = true,
   buttonClasses = '',
-  foreignKeyAccess = false,
-  showRedirection = false,
+
+  // from ToolJet database dashboard
+  tjdb = {
+    isLoading: false,
+    topPlaceHolder: '',
+    foreignKeyAccessInRowForm: false,
+    foreignKeyAccess: false,
```

Comment:
When I read this prop name, I'm unable to figure from the drawer's perspective what it might be used for unless I look at the code. Can we find a better name?